source/_integrations/vacuum.markdown

```diff
@@ -45,6 +45,14 @@ Stop the current activity of the vacuum.
 | ---------------------- | -------- | ---------------------------------------------------------------- |
 | `entity_id`            | yes      | Only act on specific vacuum. Use `entity_id: all` to target all. |
 
+### Service `vacuum.toggle`
+
+Toggles the vacuum cleaner on/off.
+
+| Service data attribute | Optional | Description                                                      |
+| ---------------------- | -------- | ---------------------------------------------------------------- |
+| `entity_id`            | yes      | Only act on specific vacuum. Use `entity_id: all` to target all. |
+
```

Comment:
If this attribute is optional, what is the behaviour if it is omitted? Does that need to be clarified in the documentation?

Reply:
Well, I haven't check how this is handled, but `entity_id` is also optional for any other entity service, like `humidifier.toggle`